src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2763,6 +2763,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       Returns="@(ComReferenceWrappers)"
       DependsOnTargets="ResolveComReferences" />
 
+  <!--
+    ============================================================
+                                      ResolveFrameworkReferences
+
+    Overrridden by Microsoft.NET.Sdk to return
+    ResolvedFrameworkReference items in order to populate the
+    Frameworks node of the Solution Explorer in the IDE.
+  -->
+  <Target Name="ResolveFrameworkReferences" />
```

Comment:
Some other stubs have the returns that's expected. I think this helps s documentation at least. @davkean, if the project system gets the stub does it matter if it has returns? Or will it benefits same as explicit return of empty items?

Reply:
I was patterning this off of [GetTargetFrameworkProperties](https://github.com/microsoft/msbuild/blob/6c53fccfab0f1a58e8d04f8c57dac058c798dcf7/src/Tasks/Microsoft.Common.CurrentVersion.targets#L1723), which doesn't return anything.

Reply:
Ok. I don't have a strong opinion on the "doc" aspect, but I would see if there's any implication for project system functionally.

Reply:
I'm pretty sure this will be fine, but @drewnoakes can you confirm?

Reply:
This looks fine to me.

If a project has `FrameworkReference` items and this version of MSBuild but not a version of the SDK that overrides this stub, all the `FrameworkReference` items will appear as unresolved, which seems like correct behaviour.